.github/workflows/submodule_update.yml

```
@@ -0,0 +1,32 @@
+name: Submodule_update
+
+on:
+  push:
+    branches: 
+      - 'master'
+      - '2.10'
+
+jobs:
+  tarantool:
+    name: Update submodule in ee repository
+    runs-on: ubuntu-20.04-self-hosted
+    env:
+      SUBMODULE: 'tarantool'
+      REPOSITORY: "${{ secrets.EE_UPDATE_SUBMODULE_REPO }}"
+      FEATURE_BRANCH: 'TarantoolBot/update-submodule'
```

we should also update tarantool@2.10 in ee, so update branches should have different names
```suggestion
      FEATURE_BRANCH: 'bot/update-tarantool-master'
```

---

Looks like fixed.